README.md

```diff
@@ -23,6 +23,17 @@ Short is built on top of [app](https://github.com/byliuyang/app), a reusable fra
 ### Create reCAPTCHA account
 [Create ReCAPTCHA account](http://www.google.com/recaptcha/admin)
 
+Register at [ReCAPTCHA](http://www.google.com/recaptcha/admin) using the following configuration.
+
+|Field         |Value        |
+|--------------|-------------|
+|Label         |`short`      |
+|reCAPTCHA type|`reCAPTCHAv3`|
+|Domains       |`localhost`  |
+
+Once registered copy the `RECAPTCHA_SECRET_KEY` to the env file
+and set the `RECAPTCHA_SITE_KEY` env variable within the build docker image step.
```

Comment:
```suggestion
3. Set `REACT_APP_ RECAPTCHA_SITE_KEY` to `SITE KEY` environmental variable when building docker image.
```

Reply:
Slightly modified but similar meaning.  Also ensured the file it goes in was explicitly called out.